packages/node-core/src/indexer/poi/poiModel.ts

```diff
@@ -90,7 +90,7 @@ export class PlainPoiModel implements PoiInterface {
   async getLatestPoiWithMmr(): Promise<ProofOfIndex | null> {
     const result = await this.model.findOne({
       order: [['id', 'DESC']],
-      where: {mmrRoot: {[Op.ne]: null}} as any, // Types problem with sequelize, undefined works but not null
+      where: {mmrRoot: {[Op?.ne]: null}} as any, // Types problem with sequelize, undefined works but not null
```

Comment:
I don't think this is right. There might be something wrong with the mock sequelize in a test